app/Models/PaymentDetail.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Models;
+
+use App\Models\Game;
+use App\Models\Payment;
+use Illuminate\Database\Eloquent\Model;
+
+class PaymentDetail extends Model
+{
+    protected $fillable = [
+        'game_id', 
+        'quantity', 
+        'subtotal', 
+        'payment_id',
+    ];
+
+    public function game()
+    {
+        return $this->belongsTo(Game::class);
+    }
+
+    public function payment()
+    {
+        return $this->belongsTo(Payment::class);
+    }
```

Comment:
Ở đây mình không muốn tạo model cho bảng trung gian có được không?

Reply:
ý anh là không tạo model PaymentDetail này ạ

Reply:
Đúng rồi đấy bạn.

Reply:
nhưng mà ở model Payment em có cần dùng đến model PaymentDetail ạ :(

Reply:
Vẫn có cách không cần đến Model trung gian mà vẫn thao tác được đấy nhé. Bạn đọc kỹ phần ORM relationship many to many.

Reply:
Như này không sai, chỉ là vẫn có một cách khác nữa thôi.

Reply:
dạ vâng ạ